deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/deployment.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: assigner
+spec:
+  template:
+    spec:
+      containers:
+        - name: assigner
+          resources:
+            limits:
+              cpu: "3"
+              memory: 2Gi
+            requests:
+              cpu: "3"
+              memory: 2Gi
+      volumes:
+        - name: config
+          configMap:
+            name: config
```

Comment:
Move `volumeMounts` from `base` deployment to this file too?

Reply:
Moved volume mount for `/config` volume.  Should `/identity` volume be moved here too?